### DISCUSSION TITLE
Is any detail guide for post Score to Gerrit within jenkins pipeline on SonarQube LTS 8.9 version

### DISCUSSION THREAD

**Discussion by ghost** [jenkinsci/sonar-gerrit-plugin#138] [Q&A] [answered]
### Describe your use-case which is not covered by existing documentation.

Is any detail guide for post Score to Gerrit within jenkins pipeline on SonarQube LTS 8.9 version

**Comment by reda-alaoui** [comment, marked as answer]
Yes, in the README.
